setup/rhel72-linuxonecc/resources/jenkins.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Jenkins Slave
+Wants=network.target
+After=network.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+Type=simple
+User={{user}}
+Environment="USER={{user}}" \
+            "SHELL=/bin/bash" \
+            "HOME=/home/{{user}}" \
+            "PATH=/usr/lib64/ccache:/usr/bin" \
+            "NODE_COMMON_PIPE=/home/{{user}}/test.pipe" \
```

Comment:
node_test_dir and jobs would be nice (i would use ansibles facts to get the number of cores since its tricky to call system commands in systemd scripts -- ref getconf) https://github.com/nodejs/build/issues/291